ghcup-prereleases-0.0.7.yaml

```diff
@@ -636,6 +636,93 @@ ghcupDownloads:
+          Linux_Debian:
+            (>= 10 && < 11): &ghc-961alpha3-64-deb10
+              dlHash: fa0c2b8e31ea182f5111feeec3c3fcff2074857b5a068d8902519fdcc96b26e3
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-deb10-linux.tar.xz
+            < 10: &ghc-961alpha3-64-deb9
+              dlHash: 9a43cad78680d6a47e338ae5bc13deada164d7203996f7791e6957945c5aa5f3
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-deb9-linux.tar.xz
+            '>= 11': &ghc-961alpha3-64-deb11
+              dlHash: d552b709426dd8e1afa499789bd9ac775c1f062005656b4c41d28bb338d6692e
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-deb11-linux.tar.xz
+            unknown_versioning: *ghc-961alpha3-64-deb11
+          Linux_Fedora:
+            '>= 33': &ghc-961alpha3-64-fedora33
+              dlHash: ade217760de958e6ed8693b16245bd244a2ccf54c262b37222d784b839d8a699
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc-961alpha3-64-centos7
+          Linux_Mint:
+            < 20: *ghc-961alpha3-64-deb9
+            '>= 20': *ghc-961alpha3-64-deb10
+          Linux_RedHat:
+            unknown_versioning: *ghc-961alpha3-64-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 19 ): *ghc-961alpha3-64-deb9
+            unknown_versioning: *ghc-961alpha3-64-deb10
+          Linux_UnknownLinux:
+            unknown_versioning: *ghc-961alpha3-64-fedora33
+          Windows:
+            unknown_versioning:
+              dlHash: a40ab69c7dde09fa0e1afa3e45a3a7d307637a116a4a1ab0a34deda330dcf627
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-mingw32
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-unknown-mingw32.tar.xz
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlHash: d6d44b25e2112ef0f86c936af97513c1cf398a5bdd6f112361abf5da3e9b990a
+              dlSubdir: ghc-9.6.0.20230210-aarch64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-aarch64-apple-darwin.tar.xz
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: 00d5d68e4795629f6279796bba4ae8281361c6ca78b574a0563703218ba27c8b
+              dlSubdir: ghc-9.6.0.20230210-aarch64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-aarch64-deb10-linux.tar.xz
+      viChangeLog: https://downloads.haskell.org/~ghc/9.6.0.20230210/docs/users_guide/9.6.1-notes.html
+      viSourceDL:
```

Comment:
We also want `viTestDL`

Reply:
Alright, I'll write this down in the release checklist.